.github/workflows/tests.yml

```
@@ -112,6 +112,17 @@ jobs:
       fail-fast: false
     steps:
       - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: pip
+          cache-dependency-path: requirements-tests.txt
+      - run: |
+          pip install tomli typing_extensions
```

```suggestion
          pip install pathspec tomli typing_extensions
```
Though maybe we should just rely on `requirements-tests.txt` instead?